apps/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -19,17 +17,9 @@ export class ListItemComponent {
   @Input() id!: number;
   @Input() name!: string;
   @Input() type!: CardType;
```

Comment:
same here, all those inputs are not necessary. This component is just a container where you want to project content. (see `ng-content`) and this cimponent will only emit event

Reply:
done